include/boost/spirit/include/qi_expectd.hpp

```diff
@@ -0,0 +1,19 @@
+/*=============================================================================
+    Copyright (c) 2001-2011 Joel de Guzman
+    Copyright (c) 2001-2011 Hartmut Kaiser
+    Copyright (c) 2016      Frank Hein, maxence business consulting gmbh
+    http://spirit.sourceforge.net/
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+=============================================================================*/
+#ifndef BOOST_SPIRIT_INCLUDE_QI_EXPECTD
+#define BOOST_SPIRIT_INCLUDE_QI_EXPECTD
```

Comment:
I could and I would. I actually don't have a clue what you want me to do neither what the problem in particular is.

Is it that a header qi_expectd.hpp in include is not acceptable? We could just remove it. As far as I understand it, the current header structure is in home, and include headers are there for historical reasons forwarding to home. If that was true, i would remove the header and adjust the doc. Is it that what I should do?

Or should I forward include both directive and operator header from qi_expect.hpp? I could do that. This would make qi_expectd obsolete without dropping the forward include. 
I assume this to be the best solution because, so I will start by committing that.

Should I change the expect operators implementation? And how? Should it use the expect directive for the parsing job of the operator? IMO there is no benefit from that which would justify the change. And would introduce a new dependency.

Or is it at last the only the name of the guardian which is not ok?

Can you please point me to the naming conventions you referred to? "The Porting from 1.8" in the docs is the only I found mentioning naming conventions specific to qi. And, of course, the boost conventions which do not go into the detail of the library's internals.